examples/tree_2d_dgsem/elixir_navierstokes_convergence.jl

```diff
@@ -170,7 +170,8 @@ end
 initial_condition = initial_condition_navier_stokes_convergence_test
 
 # BC types
-velocity_bc_top_bottom = NoSlip((x, t, equations) -> initial_condition_navier_stokes_convergence_test(x, t, equations)[2:3])
+velocity_bc_top_bottom = NoSlip((x, t, equations) -> SVector(initial_condition_navier_stokes_convergence_test(x, t, equations)[2], 
+                                                             initial_condition_navier_stokes_convergence_test(x, t, equations)[3]))
```

Comment:
This calls the IC twice (and is quite long). What do you think about something like

```suggestion
velocity_bc_top_bottom = NoSlip() do x, t, equations
    u = initial_condition_navier_stokes_convergence_test(x, t, equations)
    return SVector(u[2], u[3])
end
```